code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,518 @@
 var/list/sacrificed = list()
 
-/obj/effect/rune
+/*
+
+This file contains runes.
+Runes are used by the cult to cause many different effects and are paramount to their success.
+They are drawn with an arcane tome in blood, and are distinguishable to cultists and normal crew by examining.
+Fake runes can be drawn in crayon to fool people.
+Runes can either be invoked by one's self or with many different cultists. Each rune has a specific incantation that the cultists will say when invoking it.
 
+*/
 
-///Teleport: Teleports the user to a random teleport rune with the same rune. Can be imbued into a talisman.
-
-
-/obj/effect/rune/proc/teleport(key)
-	var/mob/living/user = usr
-	var/allrunesloc[]
-	allrunesloc = new/list()
-	var/index = 0
-	//	var/tempnum = 0
-	for(var/obj/effect/rune/R in world)
-		if(R == src)
-			continue
-		if(R.word1 == wordtravel && R.word2 == wordself && R.word3 == key && R.z != ZLEVEL_CENTCOM)
-			index++
-			allrunesloc.len = index
-			allrunesloc[index] = R.loc
-	if(index >= 5)
-		user << "<span class='danger'>You feel pain, as rune disappears in reality shift caused by too much wear of space-time fabric</span>"
-		if (istype(user, /mob/living))
-			user.take_overall_damage(5, 0)
+/obj/effect/rune
+	name = "rune"
+	var/cultist_name = "basic rune"
+	desc = "An odd collection of symbols drawn in what seems to be blood."
+	var/cultist_desc = "A basic rune with no function." //This is shown to cultists who examine the rune in order to determine its true purpose.
+	anchored = 1
+	icon = 'icons/obj/rune.dmi'
+	icon_state = "1"
+	unacidable = 1
+	layer = TURF_LAYER
+	color = rgb(255,0,0)
+
+	var/invocation = "Aiy ele-mayo!" //This is said by cultists when the rune is invoked.

```

Comment:
that's now even how you pronounce it
it's "eye-la-mao"

correct it or pick a better joke or something else.